specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/ForceDeleteVirtualMachine

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
It seems that this file does not have an extension in its name? And the content is identical to `specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/ForceDeleteVirtualMachine.json`, could we please remove this file?